asreview/io/utils.py

```diff
@@ -121,10 +121,11 @@ def standardize_dataframe(df, column_spec={}):
         else:
             try:
                 df['record_id'] = pd.to_numeric(df['record_id'])
+                df['record_id'] = df['record_id'].astype(int)
```

Comment:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_numeric.html#pandas.to_numeric 

Use downcast argument?

Reply:
I tried that, but it didn't work...

Reply:
Oke.